src/frontend/hooks/persistedState/usePersistedSettings.ts

```diff
@@ -1,25 +1,29 @@
 import {type StateCreator} from 'zustand';
 import {createPersistedState} from './createPersistedState';
-import {CoordinateFormat} from '../../sharedTypes';
+import {CoordinateFormat, SyncSetting} from '../../sharedTypes';
 
 type SettingsSlice = {
   coordinateFormat: CoordinateFormat;
   manualCoordinateEntryFormat: CoordinateFormat;
+  syncSetting: SyncSetting;
```

Comment:
the naming of this setting feels kind of vague. this is specifically related to media sync, so maybe rename that (plus any relevant types and variables) to be clearer about that? e.g. `mediaSyncMode`

Reply:
Changed to mediaSyncSetting... Hope that is ok